build/main.yml

```diff
@@ -145,19 +145,12 @@ jobs:
     - task: DockerInstaller@0
       inputs:
         dockerVersion: '17.09.0-ce'
-    - task: Bash@3
-      displayName: Install dependencies
-      inputs:
-        targetType: 'inline'
-        script: |
-          # Write your commands here
-          sudo apt install gcc-9-aarch64-linux-gnu g++-9-aarch64-linux-gnu
```

Comment:
Not required, as we're using a docker container
this is a result of copy n paste